docs/topics/native/native-memory-manager.md

```diff
@@ -31,15 +31,34 @@ and waits for its completion.
 
 ### Monitor GC performance
 
-No special instruments are currently available to monitor the GC performance. However, it's possible to look through GC
-logs to diagnose issues. To enable logging, set the following compilation flag in the Gradle build script:
+To monitor the GC performance, you can look through its logs and diagnose issues. To enable logging,
+set the following compiler option in the Gradle build script:
 
 ```none
 -Xruntime-logs=gc=info
 ```
 
 Currently, the logs are only printed to `stderr`.
 
+On Apple platforms, you can take advantage of the Xcode Instruments toolkit to debug iOS app performance.
+The garbage collector reports pauses with signposts available in Instruments.
+Signposts allow for custom logging within your app, so you can check if a GC pause corresponds to the application freeze.
```

Comment:
Suggestion (up to you):

Signposts enable custom logging within your app, allowing you to check if a GC pause causes an application freeze.